sklearn/linear_model/tests/test_base.py

```diff
@@ -105,12 +105,14 @@ def test_raises_value_error_if_positive_and_sparse():
         reg.fit(X, y)
 
 
-def test_raises_value_error_if_sample_weights_greater_than_1d():
+def test_raises_value_error_if_sample_weights_greater_than_1d(random_state=0):
     # Sample weights must be either scalar or 1D
 
     n_sampless = [2, 3]
     n_featuress = [3, 2]
 
+    rng = check_random_state(random_state)
```

Comment:
Setting this as local variable now, because this is the only test that still needs it.

Reply:
```suggestions
    rng = np.random.RandomState(0)
```

Reply:
Please see my comment about reintroducing `rng = check_random_state(0)` als global. 

Before this PR, `check_random_state` was already used in a few places. Should we use it for the global seed too, or be more explicit with `rng = np.random.RandomState(0)`?